lang/cs/Org.Apache.REEF.Client/Common/DriverFolderPreparationHelper.cs

```diff
@@ -84,6 +90,11 @@ internal void PrepareDriverFolder(AppParameters appParameters, string driverFold
             // Add the appParameters into that folder structure
             _fileSets.AddJobFiles(appParameters);
 
+            // Add the reef-bridge-client jar to the global files in the manner of JavaClientLauncher.cs.
+            _fileSets.AddToLocalFiles(Directory.GetFiles(JarFolder)
+                .Where(file => (!string.IsNullOrWhiteSpace(file)))
```

Comment:
Ok let's keep it for consistency sake, but please remove these extra parenthesis :)